extensions/solana-explorer/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "solana-explorer",
+  "title": "Solana Explorer",
+  "description": "Look up transactions and accounts on the various Solana clusters.",
+  "icon": "command-icon.png",
+  "author": "darklong",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "solana-explorer",
+      "title": "Solana Explorer",
+      "subtitle": "Look up transactions and accounts on the Solana clusters.",
```

Comment:
```suggestion
```
I think we should remove subtitle, It's usually this is the service or domain that your command is associated with.